tests/failpoints/cases/test_coprocessor.rs

```diff
@@ -421,3 +424,50 @@ fn test_read_index_lock_checking_on_follower() {
         resp
     );
 }
+
+#[cfg(feature = "failpoints")]
+#[test_case(test_raftstore::new_server_cluster)]
+#[test_case(test_raftstore_v2::new_server_cluster)]
+fn test_follower_buckets() {
+    let mut cluster = new_server_cluster(0, 3);
```

Comment:
```suggestion
    let mut cluster = new_cluster(0, 3);
```

Should use `new_cluster` otherwise it only test raftstore v1.

Reply:
none, I will use macro to impl it